.github/CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 # Release 0.14.0-dev (development release)
 
 <h3>New features since last release</h3>
+Added new Jax device.
```

Comment:
So modest :laughing:. We usually like to include a bit more here, e.g., a short code example on how to define and use the device (and what this feature means to the user).

Also, don't forget to add yourself to the list of contributors lower down!